content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -49,6 +49,28 @@ but not production. Like all alpha features, this guide is subject to change.
     $ mkdir -p "${WORK_DIR}"/"${CLUSTER_NAME}"/"${SERVICE_NAMESPACE}"
     {{< /text >}}
 
+## Workaround for a regression if using Istio 1.7.0 not present in Istio 1.6
+
+{{< tip >}}
+A regression discovered in the Makefile delivered with 1.7.0 described in this document requires
+executing steps within this section. This is fixed 1.7.1 and later versions.
+{{< /tip >}}
+
+1. Download and decompress the Istio 1.6.8 release to gain access to the Makefile used in this document.
+
+    {{< text bash >}}
+    $ pushd "${WORK_DIR"}
+    $ curl -LO https://github.com/istio/istio/releases/download/1.6.8/istio-1.6.8-linux-amd64.tar.gz
+    $ tar -zxf istio-1.6.8-linux-amd64.tar.gz
+    $ popd
+    {{< /text >}}
+
+1. Set the Istio variable used within this document.
+
+    {{< text bash >}}
+    $ ISTIO_DIR="${WORK_DIR}"/istio-1.6.8
+    {{< text >}}
+
```

Comment:
Other than this section, does the rest of the doc stay the same in 1.7.1+ (i.e., just skip this part)? If not, then this whole doc is just a 1.7.0 hack, so why not just add these commands into the mainline instructions. If the rest of the doc will stay the same, then I would move this section up and put it in a warning `{{< warning >}} For release 1.7.0 only, download the Makefile from release 1.6.8 before proceeding: <download commands> {{< /warning >}}`.

Reply:
I think we have it fixed in 1.7.0 given the Tuesday slip. Although I have not had a chance to verify that.

Reply:
@frankbu as sloppy as this is, we are having two makefiles in the release - approved by T&R, env, release managers, and leads meeting :) https://github.com/istio/istio/pull/26513